CaptureSDK/GiniCaptureSDKExample/Example Swift/ScreenAPICoordinator.swift

```diff
@@ -109,12 +109,13 @@ extension ScreenAPICoordinator: UINavigationControllerDelegate {
 				extractionAmount = ExtractionAmount(value: Decimal(string: String(amountValue.split(separator: ":")[0])) ?? 0.0, currency: .EUR)
 			}
 		
-			visionConfiguration.cleanup(paymentRecipient: extractedResults.first(where: { $0.name == "paymentRecipient"})?.value ?? "",
+			visionConfiguration.sendTransferSummary(paymentRecipient: extractedResults.first(where: { $0.name == "paymentRecipient"})?.value ?? "",
```

Comment:
[suggestion]: Indentation issue `visionConfiguration.sendTransferSummary`.
Please check also the indentation in the entire method.